variants/roboheart_hercules/pins_arduino.h

```diff
@@ -0,0 +1,87 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        20
+#define NUM_ANALOG_INPUTS       16
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<40)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 34)
+
+
+// Motor driver pins 
+#define MOTOR_A_IN1 25  //  PHASE/IN1
+#define MOTOR_A_IN2 26  //  ENABLE/IN2
+
+#define MOTOR_B_IN1 27  //  PHASE/IN1
+#define MOTOR_B_IN2 32  //  ENABLE/IN2
+
+#define MOTOR_C_IN1 33  //  PHASE/IN1
+#define MOTOR_C_IN2 4   //  ENABLE/IN2
+
+#define SLEEP_MOTOR_ABC 2  //  nSLEEP 
+
+#define LED_ROBOHEART 14  // Built in LED
```

Comment:
You can add this to make the board compatible with some examples using LED_BUILTIN :)
```
#define BUILTIN_LED  LED_ROBOHEART // backward compatibility
#define LED_BUILTIN LED_ROBOHEART
```